lambdas/services/pds_api_service.py

```diff
@@ -26,7 +27,7 @@ def handle_response(self, response: Response, nhs_number: str) -> PatientDetails
             )
 
         if response.status_code == 400:
-            raise InvalidResourceIdException("Invalid NHS number.")
```

Comment:
Either remove all `.` in logs or put them back in

Reply:
Done